doc/releases/changelog-dev.md

```diff
@@ -327,6 +327,8 @@ The Operator class has undergone a major refactor with the following changes:
   for developers.
   [(#2064)](https://github.com/PennyLaneAI/pennylane/pull/2064)
 
+* The ``operation.py`` file got cleaned up.
```

Comment:
Suggest making this more descriptive :) Or, potentially even removing it altogether if it is just a reorganization